presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/jmx/StatisticsAwareJdbcClient.java

```diff
@@ -229,6 +230,18 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
         return stats.getGetTableStatistics().wrap(() -> delegate().getTableStatistics(session, handle, tupleDomain));
     }
 
+    @Override
+    public boolean supportsLimit()
+    {
+        return stats.getSupportsLimit().wrap(() -> delegate().supportsLimit());
```

Comment:
supportsLimit stats are not useful

this method does not take any args, so it is not possible that it is not O(1) call

i would rather not have these stats, not to litter the JMX state